apps/passport-server/src/routing/routes/accountRoutes.ts

```diff
@@ -85,6 +86,27 @@ export function initAccountRoutes(
     res.status(200).json(result);
   });
 
+  app.post("/account/one-click-login", async (req: Request, res: Response) => {
```

Comment:
should this route `res.send` something?

Reply:
it does - see userService.handleOneClickLogin() in the last line of this function